services_app/src/main/java/org/opendatakit/services/sync/actions/activities/AbsSyncBaseActivity.java

```diff
@@ -535,21 +541,32 @@ public PropertiesSingleton getProps() {
     }
 
     public static void showAuthenticationErrorDialog(final Activity activity, String message) {
-        AlertDialog.Builder builder = new AlertDialog.Builder(activity);
-        builder.setTitle(R.string.authentication_error);
-        builder.setMessage(message);
-        builder.setNeutralButton(android.R.string.ok, new DialogInterface.OnClickListener() {
-            public void onClick(DialogInterface dialog, int id) {
-                if (activity instanceof VerifyServerSettingsActivity) {
-                    activity.finish();
-                }
-                dialog.dismiss();
-            }
-        });
-        AlertDialog dialog = builder.create();
-        dialog.show();
+
+        /**
+         * New dialog styling
+         * MaterialAlertDialogBuilder is standard for all ODK-X Apps
+         * OdkAlertDialogStyle present in AndroidLibrary is used to style this dialog
+         * @params
+         */
+
+        AlertDialog alertDialog = new MaterialAlertDialogBuilder(activity,R.style.OdkXAlertDialogStyle)
```

Comment:
Why isn't the type simply MaterialAlertDialogBuilder instead of androidx.appcompat.app.AlertDialog?